docs/source/ADVANCED_DOCUMENTATION.rst

```diff
@@ -0,0 +1,100 @@
+.. _ADVANCED_DOCUMENTATION:
+
+Advanced Documentation
+======================
+
+Welcome to the advanced documentation for DeepCell Kiosk developers. We will go over cluster customization, accessing cluster logs and metrics, less-common deployment workflows, a few design decisions that may be of interest to other developers, and other topics.
+
+.. contents:: :local:
+
+Preliminaries
+-------------
+
+Kiosk Setup Parameters
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. todo::
+
+    Write a description of the parameters and their defaults
```

Comment:
@willgraf Can you help covering guidelines for any parameters that need to be specified during kiosk setup?